src/main/resources/InstructorSampleData.json

```diff
@@ -2763,24 +2763,112 @@
+	  "giverSection": "Tutorial Group 1",
+	  "receiverSection": "Tutorial Group 1",
+	  "feedbackResponseId": "2%alice.b.tmms@gmail.tmt%alice.b.tmms@gmail.tmt",
+	  "showCommentTo": [
+	    "GIVER",
+	    "RECEIVER",
+	    "RECEIVER_TEAM_MEMBERS",
+	    "INSTRUCTORS"
+	  ],
+	  "showGiverNameTo": [
+	    "GIVER",
+	    "RECEIVER",
+	    "RECEIVER_TEAM_MEMBERS",
+	    "INSTRUCTORS"
+	  ],
+	  "isVisibilityFollowingFeedbackQuestion": false,
+	  "createdAt": "2017-06-03 9:17 PM +0530",
+	  "commentText": {
+	    "value": "<p>Alice, good to know that you liked applying software engineering skills in the project. Don’t forget to use the project to practice communication skills too.</p>"
+	  },
+	  "lastEditorEmail": "teammates.demo.instructor@demo.course",
+	  "lastEditedAt": "2017-06-03 9:17 PM +0530"
+	},
+	"comment1FromT1C1ToR1Q5S1C1": {
+	  "courseId": "demo.course",
+	  "feedbackSessionName": "Session with different question types",
+	  "feedbackQuestionId": "5",
+	  "giverEmail": "teammates.demo.instructor@demo.course",
+	  "giverSection": "Tutorial Group 1",
+	  "receiverSection": "Tutorial Group 1",
+	  "feedbackResponseId": "5%alice.b.tmms@gmail.tmt%benny.c.tmms@gmail.tmt",
+	  "showCommentTo": [
+	    "GIVER",
+	    "RECEIVER",
+	    "INSTRUCTORS"
+	  ],
+	  "showGiverNameTo": [
+	    "GIVER",
+	    "RECEIVER",
+	    "INSTRUCTORS"
+	  ],
+	  "isVisibilityFollowingFeedbackQuestion": false,
+	  "createdAt": "2017-06-03 9:20 PM +0530",
+	  "commentText": {
+	    "value": "<p>Completely agree, the application does look pretty.</p>"
+	  },
+	  "lastEditorEmail": "teammates.demo.instructor@demo.course",
+	  "lastEditedAt": "2017-06-03 9:20 PM +0530"
+	},
+	"comment1FromT1C1ToR3Q5S1C1": {
+	  "courseId": "demo.course",
+	  "feedbackSessionName": "Session with different question types",
+	  "feedbackQuestionId": "5",
+	  "giverEmail": "teammates.demo.instructor@demo.course",
+	  "giverSection": "Tutorial Group 1",
+	  "receiverSection": "Tutorial Group 2",
+	  "feedbackResponseId": "5%alice.b.tmms@gmail.tmt%francis.g.tmms@gmail.tmt",
+	  "showCommentTo": [
+	    "GIVER",
+	    "RECEIVER",
+	    "INSTRUCTORS"
+	  ],
+	  "showGiverNameTo": [
+	    "GIVER",
+	    "RECEIVER",
+	    "INSTRUCTORS"
+	  ],
+	  "isVisibilityFollowingFeedbackQuestion": false,
+	  "createdAt": "2017-06-03 9:23 PM +0530",
+	  "commentText": {
+	    "value": "<p>Alice, He's one of the best designers in our institute and always amazes everyone with his work.</p>"
+	  },
+	  "lastEditorEmail": "teammates.demo.instructor@demo.course",
+	  "lastEditedAt": "2017-06-03 9:23 PM +0530"
+	},
+	"comment1FromT1C1ToR2Q2S1C1": {
+	  "courseId": "demo.course",
+	  "feedbackSessionName": "Session with different question types",
+	  "feedbackQuestionId": "2",
+	  "giverEmail": "teammates.demo.instructor@demo.course",
+	  "giverSection": "Tutorial Group 2",
+	  "receiverSection": "Tutorial Group 2",
+	  "feedbackResponseId": "2%charlie.d.tmms@gmail.tmt%charlie.d.tmms@gmail.tmt",
```

Comment:
Hmm, looks like I remembered the default data wrongly. I always thought names A to D were in Team 1 =P